app/views/dashboard_v2/_sidebar.html.erb

```diff
@@ -47,7 +48,9 @@
               <% end %>
           </div>
         </div>
-        <%# TODO Search topics %>
+        <%# Search topics %>
+        <br/><br/>
```

Comment:
```suggestion
        <h3>Search topics</h3>
```

What if we did this, and also converted other instances of `<%# ... %>` into regular HTML comments like `<!-- ... -->`? I think that could be nice to leave the rest of the page just normal HTML and not insert Ruby just to make a comment. Not a big deal, but stylistically nice!